### PR TITLE
Add anisotropy tests to render-fidelity-tests (and update Babylon + Filament to latest)

### PR DESCRIPTION
**Pull request by bhouston**
This builds upon my previous PR that upgrades from glTF-Sample-Models to glTF-Sample-Assets: https://github.com/google/model-viewer/pull/4534

This PR pulls in additional tests from the new glTF-Sample-Asset repository, specifically 3 new anisotropy tests.

I have also upgraded Babylon.JS and Filament to their latest versions and updated their goldens as well as model-viewer's goldens.

### CODE REVIEW

**Comment by bhouston** [comment]
Fixes #4486

**Comment by bhouston** [comment]
@elalish I removed the anisotropyBarnLamp test so that we pass the fidelity test suite.  So I believe this is ready to merge.

File: packages/render-fidelity-tools/test/goldens/khronos-AnisotropyRotationTest/three-gpu-pathtracer-golden.png
Comment (elalish) [inline packages/render-fidelity-tools/test/goldens/khronos-AnisotropyRotationTest/three-gpu-pathtracer-golden.png:1]: Should we remove this example from three-gpu-pathtracer? In future we should probably evaluate if three-gpu-pathtracer is stable enough to keep in these tests...

**Review by elalish — DISMISSED** [review]
I take it this takes the place of #4534? Not a big deal, but it's a bit easier to review these things if you just push changes to that branch/PR instead of creating a new one.

**Comment by bhouston** [comment]
@elalish wrote:
> I take it this takes the place of https://github.com/google/model-viewer/pull/4534? Not a big deal, but it's a bit easier to review these things if you just push changes to that branch/PR instead of creating a new one.

Technically each of these PR's build upon the previous.  So if you wanted to merge https://github.com/google/model-viewer/pull/4534 first as it is good.

Then we can modify this one, and then once it is in, we can then discuss the next one that adds more models: https://github.com/google/model-viewer/pull/4537

File: packages/render-fidelity-tools/test/goldens/khronos-AnisotropyRotationTest/three-gpu-pathtracer-golden.png
Comment (bhouston) [reply packages/render-fidelity-tools/test/goldens/khronos-AnisotropyRotationTest/three-gpu-pathtracer-golden.png:1]: I've removed it as you requested.

Comment (elalish) [reply packages/render-fidelity-tools/test/goldens/khronos-AnisotropyRotationTest/three-gpu-pathtracer-golden.png:1]: Are you sure?

File: packages/render-fidelity-tools/package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "@types/puppeteer": "^5.4.6",
     "@types/rimraf": "^3.0.1",
     "@types/three": "^0.156.0",
-    "filament": "1.31.5",
+    "filament": "1.44.0",
```
Comment (elalish) [inline packages/render-fidelity-tools/package.json:39]: Did none of the existing renders change appreciably when these versions were bumped?

Comment (elalish) [reply packages/render-fidelity-tools/package.json:39]: Okay, lots of PRs merged - let's see how this does when it's all pulled together!

File: packages/render-fidelity-tools/test/goldens/khronos-AnisotropyRotationTest/three-gpu-pathtracer-golden.png
Comment (elalish) [inline packages/render-fidelity-tools/test/goldens/khronos-AnisotropyRotationTest/three-gpu-pathtracer-golden.png:1]: probably exclude the blank ones?

**Review by elalish — APPROVED** [review]
This looks great, thanks! @gkjohnson what's the status with three-gpu-pathtracer these days? Are you planning to upkeep it?

**Comment by gkjohnson** [comment]
> This looks great, thanks! @gkjohnson what's the status with three-gpu-pathtracer these days? Are you planning to upkeep it?

I've tried to but browser vendors (mostly Apple and Safari) keep adding api regressions that cause it to break after fixes have been added 😓 The recent  blocker is that Apple's recent OS update introduced issues that cause the path tracer to crash _hard_ so I can barely test it at the moment since I don't have easy access to my other machine. The bug is being tracked [here](https://bugs.chromium.org/p/angleproject/issues/detail?id=8396).

I'm going to wait this out a bit to see what happens with this fix before diving into some of the new features too much more. Some others are helping to improve the renering on iOS so the project is still moving - but at the moment it's finding a few browser bugs 😅

**Comment by elalish** [comment]
@gkjohnson Gotcha, thanks. I'm having trouble even on Chrome, I think because of updating three.js? Do you want to take a look at why we seem to just get blank renders now?

**Comment by bhouston** [comment]
@elalish could we just commit this as is?  Bugs in three-gpu-pathtracer is a bit orthogonal to his PR.  And what is great is that @gkjohnson can use the new tests in this PR to further improve his renderer.

File: packages/render-fidelity-tools/test/goldens/khronos-AnisotropyRotationTest/three-gpu-pathtracer-golden.png
Comment (bhouston) [reply packages/render-fidelity-tools/test/goldens/khronos-AnisotropyRotationTest/three-gpu-pathtracer-golden.png:1]: removed!

**Review by elalish — APPROVED** [review]
Agreed, thanks!

**Comment by gkjohnson** [comment]
> @gkjohnson Gotcha, thanks. I'm having trouble even on Chrome, I think because of updating three.js? Do you want to take a look at why we seem to just get blank renders now?

Without knowing your platform it's hard to say. On Mac these issues are operating system-level driver issues that affect all browsers recently introduced with MacOS 14.0. But like I said I can't run things on my machine on Safari or Chrome due to these system regressions.

**Pull request merged**